### PR TITLE
Adds ability to easily modify head tags

### PR DESCRIPTION
**Pull request by aendrew**

### CODE REVIEW

File: stories/__snapshots__/Storyshots.test.js.snap [hunk 1]
```diff
@@ -498,7 +498,7 @@ exports[`Storyshots Analytics default 1`] = `
     <noscript
       data-o-component="o-tracking"
     >
-      &lt;img src="https://spoor-api.ft.com/px.gif?data=%7B%22action%22%3A%22view%22%2C%22category%22%3A%22page%22%2C%22context%22%3A%7B%22content%22%3A%7B%22asset_type%22%3A%22interactive%22%2C%22uuid%22%3A%22xxx-xxx-xxx%22%7D%2C%22product%22%3A%22IG%22%7D%2C%22system%22%3A%7B%22is_live%22%3Afalse%2C%22apiKey%22%3A%22qUb9maKfKbtpRsdp0p2J7uWxRPGJEP%22%2C%22source%22%3A%22o-tracking-ns%22%2C%22version%22%3A%221.0.0%22%7D%7D" height="1" width="1" /&gt;
+      &lt;img src="https://spoor-api.ft.com/px.gif?data=%7B%22action%22%3A%22view%22%2C%22category%22%3A%22page%22%2C%22context%22%3A%7B%22content%22%3A%7B%22asset_type%22%3A%22interactive%22%7D%2C%22product%22%3A%22IG%22%7D%2C%22system%22%3A%7B%22is_live%22%3Afalse%2C%22apiKey%22%3A%22qUb9maKfKbtpRsdp0p2J7uWxRPGJEP%22%2C%22source%22%3A%22o-tracking-ns%22%2C%22version%22%3A%221.0.0%22%7D%7D" height="1" width="1" /&gt;
```
Comment (joannaskao) [inline stories/__snapshots__/Storyshots.test.js.snap]: Was the UUID supposed to go away here?

File: components/html-head/index.js [hunk 2]
```diff
@@ -22,13 +22,15 @@ class HtmlHead extends PureComponent {
...
     const {
+      children,
       flags,
       dataMeta,
       description,
       facebookDescription,
       facebookHeadline,
       facebookImage,
       headline,
+      headTags,
```
Comment (joannaskao) [inline components/html-head/index.js]: Can we rename this prop to something like `includeHeadTags` or something that sounds more boolean-y?

File: stories/__snapshots__/Storyshots.test.js.snap (hunk 1 above)
Comment (aendrew) [reply stories/__snapshots__/Storyshots.test.js.snap]: No; this is the testing UUID and it should be the default. I had this change on my master branch when I took a look at this just now so figured I'd include it. It's unrelated.

File: components/html-head/index.js (hunk 2 above)
Comment (aendrew) [reply components/html-head/index.js]: Good shout, one second.

File: stories/__snapshots__/Storyshots.test.js.snap (hunk 1 above)
Comment (joannaskao) [reply stories/__snapshots__/Storyshots.test.js.snap]: 👍

**Pull request merged**